sites/all/themes/humanitarianresponse/assets/sass/partials/_panel.scss

```diff
@@ -109,6 +109,22 @@
   }
 }
 
+//Events hid-contact-ref block
+.panel-pane.pane-node-field-hid-contact-ref {
```

Comment:
Let's not make this dependant of panels, whenever this field is displayed should have a consistent look